src/Rules/StarterWeb/AI/IndividualAuth/appsettings.json

```diff
@@ -2,11 +2,7 @@
   "ApplicationInsights": {
     "InstrumentationKey": ""
   },
-  "Data": {
-    "DefaultConnection": {
-      "ConnectionString": "Server=(localdb)\\mssqllocaldb;Database=aspnet-$safeprojectname$-$guid2$;Trusted_Connection=True;MultipleActiveResultSets=true"
-    }
-  },
+  "DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=aspnet-$safeprojectname$-$guid2$;Trusted_Connection=True;MultipleActiveResultSets=true",
```

Comment:
I understand that we had agreed previously on having the "Data" section but I think it means much less right now. 

The change that just went in Configuration is about removing the "Data:MyConnection:ConnectionString" convention we used to have and map to Azure Web Site connection strings directly to settings with the same name. See https://github.com/aspnet/Configuration/issues/376. This removes any mangling of the user provided name from this convention in gives users full freedom to structure their application settings in whatever way they want.

This also means that for customers to match "Data:ConnectionString" they will have to type exactly "Data:ConnectionString" in the Azure Web Site UI.

Personally I don't see much value in keeping the "Data" section in the default configuration in the template given that it sets precedent of a convention that is actually superfluous and unnecessary, but in theory we could keep it.

cc @Eilon @danroth27 What do you guys think? Anyone else needs to review this?